device-api/src/device.rs

```diff
@@ -203,7 +208,8 @@ impl Device {
     }
 
     /// List performance counters for each device files.
-    pub fn performance_counters(&self) -> Vec<(&DeviceFile, PerformanceCounter)> {
+    #[allow(dead_code)]
+    fn performance_counters(&self) -> Vec<(&DeviceFile, PerformanceCounter)> {
```

Comment:
This code is used in PR 81. I think we can keep it public and remove the dead_code as well.